src/STAGING_ENV_VARS.md

```diff
@@ -3,6 +3,7 @@ See `ceph-container/maint-lib/stagelib/envglobals.py` for most updated list and
 
 ## Required as input to staging
  - CEPH_VERSION
+ - CEPH_POINT_RELEASE
```

Comment:
`CEPH_P_R` is not required as input (as much as I can tell). It is generated and so belongs in a below section. I would suggest 

```
## Generated during staging
### Ceph version
 - CEPH_POINT_RELEASE

Reply:
Right.